react/features/base/util/helpers.js

```diff
@@ -65,3 +67,15 @@ export function assignIfDefined(target: Object, source: Object) {
 
     return to;
 }
+
+/**
+ * Prints the error and reports it to the global error handler.
+ *
+ * @param {Error} e - The error object.
+ * @param {string} msg - A custom message to print in addition to the error.
+ * @returns {void}
+ */
+export function reportError(e: Object, msg: string = '') {
+    logger.error(msg, e);
+    window.onerror && window.onerror(msg, null, null, null, e);
```

Comment:
I know this is coming from the initial implementation, but is there any other reason for us to want to obfuscate the source, line number, col number of the error? Why don't we get those from the error object and pass them out to the window.onerror?

Reply:
I don't think you can get those values from the error object, at least no on all browsers. Reading on MDN and playing with the Error object in console, the error object does not have those values as members.

Reply:
Maybe from error.stack ?